lib/zendesk_apps_tools/command.rb

```diff
@@ -90,33 +90,42 @@ def clean
     DEFAULT_SERVER_PATH = './'
     DEFAULT_CONFIG_PATH = './settings.yml'
     DEFAULT_SERVER_PORT = 4567
-    DEFAULT_APP_ID = 0
 
     desc 'server', 'Run a http server to serve the local app'
     method_option :path, default: DEFAULT_SERVER_PATH, required: false, aliases: '-p'
     method_option :config, default: DEFAULT_CONFIG_PATH, required: false, aliases: '-c'
     method_option :port, default: DEFAULT_SERVER_PORT, required: false
-    method_option :app_id, default: DEFAULT_APP_ID, required: false
```

Comment:
We can make it a one-app-only option. If you load more than 1 app the option will do nothing. Because we don't know to WHICH app you want to apply this, and what will be the number for the next app.

How about that?